config/pipelines/role.yaml

```diff
@@ -0,0 +1,48 @@
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+    name: meteor-external
+subjects:
+    - kind: ServiceAccount
+      name: pipeline
```

Comment:
`pipelines` serviceaccount is created by openshift-pipelines and it have scc privileges required to run the pipelines.
we should keep using it, or else we would have to create another SA with some permission.
If using default SA, it would cause confusion, as we would have to add additional rbac, that wouldn't keep it default anymore.